code/game/gamemodes/game_mode.dm

```diff
@@ -461,8 +461,7 @@
 						continue //Dead mob, ghost abandoned
 				else
 					if(D.can_reenter_corpse)
-						msg += "<b>[L.name]</b> ([ckey(D.mind.key)]), the [L.job] (<span class='boldannounce'>This shouldn't appear.</span>)\n"

```

Comment:
What about removing the error report for admins? Just incase there is a possible bug somehow down the line.